tools/scripts/acr-build.ps1

```diff
@@ -27,11 +27,6 @@ Param(
     [switch] $Debug
 )
 
-# use registry from environment if not passed in
-if ([string]::IsNullOrEmpty($Registry) -and ![string]::IsNullOrEmpty($env:azureContainerRegistry)) {
-    $Registry = $env:azureContainerRegistry
-}
-
 # Check path argument and resolve to full existing path
 if ([string]::IsNullOrEmpty($Path)) {
     throw "No docker folder specified."
```

Comment:
This is a bit misleading. Most users would expect a docker folder to be the folder with the Dockerfile. But this actually needs the container.json file. We should improve the error message. Consider adding sample call with required params to readme.

Reply:
not fixing this one.